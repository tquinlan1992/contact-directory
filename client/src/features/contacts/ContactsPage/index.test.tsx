import Enzyme, { shallow } from "enzyme";
import { ContactsPage } from ".";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
Enzyme.configure({ adapter: new Adapter() });

describe("ContactsPage", () => {
  it("should render the table and create contact form", () => {
    const wrapper = shallow(<ContactsPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
