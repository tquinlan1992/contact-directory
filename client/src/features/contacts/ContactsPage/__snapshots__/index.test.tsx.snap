// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactsPage should render the table and create contact form 1`] = `
<div
  className="contacts_page"
>
  <div
    className="contacts_page-create_contact"
  >
    <CreateContact />
  </div>
  <ContactsTable />
</div>
`;
